FlowCrypt/src/main/res/layout/fragment_feedback.xml

```diff
@@ -64,12 +64,12 @@
         android:id="@+id/textViewAuthorHint"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:text="@string/i_will_usually_reply_within_an_hour_except_when_i_sleep_tom"
         android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
         android:textColor="@color/gray"
         app:layout_constraintBottom_toTopOf="@+id/guidelineBottom"
         app:layout_constraintEnd_toStartOf="@+id/guidelineRight"
-        app:layout_constraintStart_toStartOf="@+id/guidelineLeft" />
+        app:layout_constraintStart_toStartOf="@+id/guidelineLeft"
+        tools:text="@string/i_will_usually_reply_within_an_hour_except_when_i_sleep_tom" />
```

Comment:
That's funny string ID, and I guess that was the actual text at some point, but now it maps to the different text:
```xml
    <string name="i_will_usually_reply_within_an_hour_except_when_i_sleep_tom">Thank you for trying FlowCrypt! We\'re here to help.</string>
```
So I suggest to rename it to the something more actual at this time.

Reply:
Agree. I will change it.